core/src/main/java/io/questdb/cairo/PartitionBy.java

```diff
@@ -298,6 +330,7 @@ public interface PartitionFloorMethod {
         fmtMonth = compiler.compile("yyyy-MM");
         fmtYear = compiler.compile("yyyy");
         fmtHour = compiler.compile("yyyy-MM-ddTHH");
+        fmtWeek = compiler.compile("YYYYWww");
```

Comment:
I do think its more consistent and possibly look more clean but I don't believe it would be following ISO Week standards. The standard goes usually like this `2022W02`, rather than `2022-W02`. Also, I would like to note `W` is being used a delimiter here instead of a formatter.

Reply:
No, such format is in ISO 8601: https://en.wikipedia.org/wiki/ISO_8601#Week_dates

Reply:
You are indeed correct. I must have overlooked the ISO standard.